tyr/security_groups/management.py

```diff
@@ -0,0 +1,35 @@
+rule = 'management'
+
+definition = {
+    'rules': [
+        {
+            'port': 22,
+            'source': [
+                'p-ops-vpn',
+                '@hudl-lincoln-east',
+                '@hudl-lincoln-west',
+                '@hudl-boston'
+                '@hudl-omaha'
+            ]
+        },
+        {
+            'port': 3389,
+            'source': [
+                'p-ops-vpn',
+                '@hudl-lincoln-east',
+            ]
+        },
+        {
+            'port': '27017 - 28019',
```

Comment:
hrm, this should probably be '27017 - 27019' + '28017 - 28019' there's 998 ports in there open that don't need to be open